### PR TITLE
Dev/mrp/week2

### PR DESCRIPTION
**Pull request by mckennaplunkett**
Assignment is ready for review.

-MP

### CODE REVIEW

**Comment by mattandccu** [comment]
Great job getting this done! I haven't done the grading yet, but I just
wanted to let you know that it builds successfully for me and all of the
tests pass. Nice work!

Matt

**Comment by mckennaplunkett** [comment]
Thanks for letting me know! I really appreciate it. Have a wonderful rest of your Sunday!

McKenna

File: Week1CodingStandards/App.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
-<configuration>
```
Comment (mattandccu) [inline Week1CodingStandards/App.config]: It looks like you inadvertently included files from Week1. You should ensure that each commit only includes files that you intend so you and other developers are not confused when looking at history. Why was this file included? Did it change it some way? Was this an accident?

File: Week2ClassHierarchy/Week2ClassHierarchy/Association.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Week2ClassHierarchy
+{
+    /// <summary>
+    /// Class for organization contacts who can be characterized as associations.
+    /// </summary>
+    public class Association : Organization
+    {
```
Comment (mattandccu) [inline Week2ClassHierarchy/Week2ClassHierarchy/Association.cs]: Consider using more MSDN style class descriptions, such as "Represents a group of people with a common purpose and having a formal structure."

File: Week2ClassHierarchy/Week2ClassHierarchy/Association.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        // Property definition for getting and setting relationship statuses between associations and the user.
+        public string RelationshipStatus
```
Comment (mattandccu) [inline Week2ClassHierarchy/Week2ClassHierarchy/Association.cs]: We don't require XML commentary on properties, but I'm glad you added comments. You should consider using XML comment style, though, so that other developers see it in intellisense. Rather than // comment, using something like:

/// <summary>
/// Gets or sets the relationship status between association members.
/// </summary>

**Comment by mattandccu** [comment]
Great job, McKenna. Your solution builds and all tests pass. I've made a few comments about things to note for future exercises.
